source/Applications/Advanced/Downsample/Downsample.cpp

```diff
@@ -52,7 +52,7 @@ Zivid::PointCloud downsample(const Zivid::PointCloud &pointCloud, int downsampli
 	point cloud, e.g. 2 - one-half,  3 - one-third, 4 one-quarter, etc.
 	*/
 
-    if((pointCloud.height() % downsamplingFactor) || (pointCloud.width() % downsamplingFactor))
+    if((pointCloud.height() % downsamplingFactor) != 0U || (pointCloud.width() % downsamplingFactor) != 0U)
```

Comment:
Is this really the correct way? :O Looks very suspicious

Reply:
It's otherwise an implicit conversion to bool from uint